src/Microsoft.Diagnostics.Monitoring.WebApi/ProfilerMessage.cs

```diff
@@ -1,17 +1,39 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if STARTUPHOOK || HOSTINGSTARTUP
+using Microsoft.Diagnostics.Monitoring.StartupHook.MonitorMessageDispatcher.Models;
+#else
+using Microsoft.Diagnostics.Monitoring.WebApi.Models;
+#endif
 using System;
 using System.Text;
 using System.Text.Json;
+using System.Threading;
 
 namespace Microsoft.Diagnostics.Monitoring
 {
+    internal sealed class EmptyPayload { }
```

Comment:
All of these new classes shouldn't be in the WebApi assembly since nothing in it uses them.